schema/resolvers.js

```diff
@@ -411,7 +411,8 @@ const Resolvers = {
     textValue: ({ type, value }) => (type === "Text" ? value : null),
     languageValue: ({ type, value }) => (type === "Language" ? value : null),
     regionValue: ({ type, value }) => (type === "Region" ? value : null),
-    dateValue: ({ type, value }) => (type === "Date" ? value : null)
+    dateValue: ({ type, value }) =>
+      type === "Date" ? (value ? new Date(value) : null) : null
```

Comment:
Is it worth splitting over a number of lines now instead of nested ternaries?

Reply:
This would be the alternative:

```
      if (type === "Date") {
        if (value) {
          return new Date(value)
        } else {
          return null;
        }
      } else {
        return null
      }
  
```

Reply:
Or:
```
if (type !== "Date" || !value) {
  return null;
}
return new Date(value);
```

Reply:
Amended.